packages/wesley-cli/src/commands/continuum-witness-report.mjs

```
@@ -0,0 +1,231 @@
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+import { WesleyError } from '@wesley/core';
+import {
+  createCheck,
```

ESLint

[error] 'createCheck' is defined but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)